assembly/pom.xml

```diff
@@ -33,6 +33,23 @@
     <assembly.name>livy-server-${project.version}</assembly.name>
   </properties>
 
+  <profiles>
+    <profile>
+      <id>targz</id>
+      <properties>
+        <assembly.format>tar.gz</assembly.format>
+      </properties>
+      <activation><property><name>targz</name></property></activation>
+    </profile>
+    <profile>
+      <id>zip</id>
```

Comment:
Isn't this whole profile the same as declaring `<assembly.format>zip</assembly.format>` in the properties section?